### ISSUE TITLE
Enable 2FA for Schlundtech API, unfortunately no access token available

### ISSUE THREAD

**Issue by thauke** [wosc/schlund-ddns#3]
hi, 
thanks for your great work!

Since i have 2fa activated i am not able to use schlund-ddns anymore.
Please add support for 2fa.

Some python library's already available, eg:
https://github.com/pyauth/pyotp

thanks in advanced,
timm

**Comment by thauke** [comment]
additional information from schlundtech support.

the token has to be in the `<auth></auth>` section:
```
<user>1226005564</user>
<password>*******</password>
<token>12345</token>
```

thanks in advanced,
timm

**Comment by wosc** [comment]
Sorry I'm so very late to this, I've just released version 1.3 where I tried to implement this feature. Care to give it a try?

**Comment by seh-len** [comment]
Hi,
it works fine for schlund-ddns but unfortunately not for schlund-ddns-cgi.
I would be very grateful if you could add support for otp tokens to the web version.
Kind regards,
Simon

**Comment by wosc** [comment]
Hi @seh-len you're right, passing the config option for totp in cgi mode was missing. I've now added it and released verion 1.6

**Comment by seh-len** [comment]
Thank you very much, especially for the fast solution.
